addons/web/static/src/scss/form_view_extra.scss

```diff
@@ -94,7 +93,7 @@
     .oe_chatter {
         display: block;
         min-width: $sheet-min-width + 2* $sheet-padding;
-        max-width: $sheet-max-width + 2* $sheet-padding;
+        max-width: $o-form-view-sheet-max-width + 2* $sheet-padding;
```

Comment:
Please try to remove the whole class encapsulating this line. "chatter" stuff has nothing to do in "web".

Isn't `oe_chatter` removed/replaced anyway?

If total remove is not possible, move relevant parts in `form_renderer.scss`.